src/cdk/a11y/live-announcer/live-announcer.ts

```diff
@@ -149,17 +159,59 @@ export class LiveAnnouncer implements OnDestroy {
     }
   }
 
-  private _createLiveElement(): HTMLElement {
-    const elementClass = 'cdk-live-announcer-element';
-    const previousElements = this._document.getElementsByClassName(elementClass);
-    const liveEl = this._document.createElement('div');
+  private _enqueueAnnouncement(
+      message: string, politeness: AriaLivePoliteness, duration?: number): Promise<void> {
+    const liveElement = this._createLiveElement();
+    // TODO: ensure changing the politeness works on all environments we support.
+    liveElement.setAttribute('aria-live', politeness);
+
+    return new Promise(resolve => {
+      this._taskQueue.enqueue(() => {
```

Comment:
Upon looking at the code, I think the `TaskQueue` isn't strictly necessary and that this could be done a bit more simply. You could instead keep two arrays, `_assertiveQueue` and `_politeQueue`. After any given announcement finishes, pop the front message from `_assertiveQueue` if not empty, then `_politeQueue`, and repeat.